andrea/js/app.js

```diff
@@ -0,0 +1,67 @@
+// buttons //
+var signUp = document.querySelector(".navbar-sign");
+var logIn = document.querySelector(".navbar-login");
```

Comment:
In this case you can use IDs as selectors as it would be even clearer what are you querying for. Also, improve the name of the variables so its even more clear what reference you are storing. Example: 
```
let logInButton = document.querySelector("#login-btn");
```